examples/mmcloud_plugin/requirements.in

```diff
@@ -0,0 +1,2 @@
+flytekitplugins-envd
```

Comment:
need to run `pip-compile` (see [here](https://docs.flyte.org/projects/cookbook/en/latest/contribute.html#updating-dependencies)) to make sure `requirements.txt` is compiled (currently it's an empty file)

Reply:
I got an error `No matching distribution found for flytekitplugins-mmcloud` when running `pip-compile` on this. I think it's because `flytekitplugins-mmcloud` has not been published yet?

Reply:
We'll need to wait for `flytekitplugins-mmcloud` to be published in the next flytekit release to merge the flytesnacks docs